app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -234,19 +234,13 @@ function initializePodcastPlayback() {
   }
 
   function muteUnmute(audio) {
-    if (audio.muted) {
-      audio.muted = false;
-      getById('mutebutt').classList.add('hidden');
-      getById('volumeindicator').classList.add('showing');
-      getById('mutebutt').classList.remove('showing');
-      getById('volumeindicator').classList.remove('hidden');
-    } else {
-      audio.muted = true;
-      getById('mutebutt').classList.add('showing');
-      getById('volumeindicator').classList.add('hidden');
-      getById('mutebutt').classList.remove('hidden');
-      getById('volumeindicator').classList.remove('showing');
-    }
+    var mutebutton = getById('mutebutt')
+    var volindicator = getById('volumeindicator')
```

Comment:
Could these be `const`s instead?

Reply:
can they also be camelCase? `muteButton` and `volumeIndicator` for example

Reply:
@jacobherrington They indeed **should** :blush:

Reply:
@rhymes thx for the guidance :+1: